app/components/coinPortfolioItem/CoinCard.tsx

```diff
@@ -50,9 +51,9 @@ const CoinCard = ({ storageData }:{ storageData:any }) => {
           <div className="flex justify-between mb-2 mt-4">
             <h2 className="text-xl">Market Price</h2>
             <div className={clsx("pt-1 pl-1 w-[30px] h-[30px] rounded-sm cursor-pointer",{
-              "bg-[#3A3978]" : darkmode,
+              "bg-[#3A3978] hover:border-cryptoblue-800 hover:border-[1px] box-border" : darkmode,
               "bg-cryptoblue-800" : !darkmode,
-            })}>
+            })} onClick={(e)=>toggleDeleteModal(e, coinData)} >
```

Comment:
do you really need to pass the event?

Reply:
You are right, there is no need for this, I will remove this